TYPO3.Flow/Classes/TYPO3/Flow/Command/ResourceCommandController.php

```diff
@@ -84,7 +84,10 @@ public function publishCommand($collection = null)
             foreach ($collections as $collection) {
                 /** @var CollectionInterface $collection */
                 $this->outputLine('Publishing resources of collection "%s"', array($collection->getName()));
-                $collection->publish();
+                $target = $collection->getTarget();
```

Comment:
If userland code calls `$collection->publish()` will there be any optimization? I guess not, would that be possible? Or is the code (in Neos, first and foremost) publishing full collections just this code here?